src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -885,7 +879,7 @@ public void ColdStartFix()
 
             // Fix Cold start for plugins
             string s = "m";
-            var pluginQueryPairs = QueryBuilder.Build(ref s);
+            var pluginQueryPairs = QueryBuilder.Build(s);
```

Comment:
Since `s` is not passed anymore as a reference, let's remove it.
Also, we should add a comment to explain what is `"m"`.

Reply:
done